test/metabase/models/params/chain_filter_test.clj

```diff
@@ -670,3 +670,25 @@
           (testing "`true` if the limit is less than the number of values the field has"
             (is (= true
                    (:has_more_values (chain-filter venues.latitude {venues.price 4} :limit 1))))))))))
+
+(deftest chain-filter-inactive-test
+  (testing "Inactive fields are not used to generate joins"
+    (mt/dataset avian-singles
+      (mt/$ids nil
```

Comment:
I see that you can skip putting `nil` here (it is implicit if the macro only gets a single form)

Reply:
No way I'm doing this, I've been burned so much by just adding a second form and then not understanding what's going on 🤣 This is the worst arity dispatching situation ever 🤣

Reply:
Makes sense, and my condolences - we should probably give the table-nameless version of this function a different name to fix that foot gun.